standard_reports/item-usage/latest/convert_data_js/codegenTypes.ts

```diff
@@ -0,0 +1,46 @@
+// Content of this file is generate but then manually pasted here, see comment in codegen.yaml
+export type Maybe<T> = T | null;
```

Comment:
not sper happy about this, but wanted to do a full typescript example

Reply:
Hm agree, though seems likes these have been fairly stable - are you sure there isn't a way that we can define this is the root of reports rather than in the individual report though? At least then would only have to update once? I guess not quite sure what that looks like for custom reports... do we have TS examples in those yet?

Reply:
I wanted it in the root of the reports, but it did mean a lot of reshuffling, and actually using adding custom reports as submodule, or export types as npm packages from core (due to the 'rootDir' nature of TS)